main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -621,4 +629,71 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
         }
     }
   }
+
+  /**
+    * Package the current project and release it on GitHub.
+    */
+  def release(flix: Flix): Validation[Unit, BootstrapError] = {
+    val formatter = flix.getFormatter
+
+    // Ensure that we have a manifest
+    val manifest = optManifest match {
+      case Some(m) => m
+      case None => return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.MissingManifest))
+    }
+
+    // Check if `github` option is present
+    val githubRepo = manifest.github match {
+      case Some(r) => r
+      case None =>
+        return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.NoLinkedProject))
+    }
+
+    // Check if `--github-key` option is present
+    val githubKey = flix.options.githubKey match {
+      case Some(k) => k
+      case None =>
+        return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.MissingApiKey))
+    }
+
+    if (!flix.options.skipPrompts) {
+      // Ask for confirmation
+      var continue = false
+      while (!continue) {
+        print(s"Release ${formatter.blue(s"github:$githubRepo")} ${formatter.yellow(s"v${manifest.version}")}? [y/n]: ")
+        val response = readLine()
+        response.toLowerCase match {
+          case "y" => continue = true
+          case "n" => return Validation.toHardFailure(BootstrapError.ReleaseError(ReleaseError.Cancelled))
+          case _ => // Continue loop
```

Comment:
I suggest to fail here. Not to re-prompt.

Reply:
Default to 'n'?